packages/ds-healthcare-gov/src/styles/settings/_healthcare-components-theme.scss

```diff
@@ -34,51 +34,153 @@ $badge__background-color--success: #12890e;
 $badge__background-color--warn: #f8c41f;
 $badge__font-color: #ffffff;
 $badge__font-color--warn: #262626;
+$button__background-color--active: #045d83;
```

Comment:
Maybe I'm missing the point here - shouldn't these values be mapped back to color tokens and not explicit hex values?

Reply:
These are generated from tokens, they are mapped in the token system.